app/frontend/app/models/group-member.js

```diff
@@ -0,0 +1,25 @@
+import DS from 'ember-data';
+import Ember from 'ember';
+
+export default DS.Model.extend({
+  user: DS.belongsTo('user'),
+  rank: DS.attr('string'),
+  pending: DS.attr('boolean'),
+  groupId: DS.attr('string'),
+
+  // This suffers the same problem as `substories`, where the
+  // relationship only serializes a subset of `group-members`,
+  // rather than all and ember will destroy the relationship.
```

Comment:
Is this a bug, should we report it, or is it already reported?  If it's already reported, we should put a link to the issue so down the road we can check the status and potentially remove the hack